lazy_tensor_core/test/cpp/test_aten_ltc_ts_tensor.cpp

```diff
@@ -1627,9 +1627,9 @@ TEST_F(AtenLtcTsTensorTest, TestNormInDimsKeep) {
 TEST_F(AtenLtcTsTensorTest, TestNormalTwoTensor) {
   at::Tensor mean = at::zeros({10, 10, 10}, at::dtype(at::kFloat));
   at::Tensor std = at::ones({10, 10, 10}, at::dtype(at::kFloat));
-  ForEachDevice([&](const Device& device) {
-    at::Tensor xla_mean = bridge::CreateLtcTensor(mean, device);
```

Comment:
what's the reason for this change? (just curious)

Reply:
Just to align the behaviors of these several tests with the remaining and to remove the additional overload of `ForEachDevice` that supports a backend device type. It's now all c10::Device.